.github/workflows/api.yaml

```
@@ -11,16 +11,27 @@
-          enable-cache: "true"
+          enable-cache: true
+          uv-version: ${{ steps.uv-version.outputs.version }}
+
       - name: Install dependencies
-        run: devbox run -- uv sync
+        run: uv sync
+          
       - name: Check dependencies
-        run: devbox run -- uv run ./scripts/check_dependencies.py
+        run: uv run ./scripts/check_dependencies.py
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)